content/news/playtest-20151114.md

```diff
@@ -0,0 +1,53 @@
+We recieved some great feedback on our [Halloween playtest](/news/playtest-20151031/), and have been hard at work over the last two weeks fixing the issues that have been reported.  Thanks to everyone who has tried the playtest and reported bugs or other feedback!
+
+We are now happy enough with the stability and integration of [all the new
+features](/news/playtest-20151031/) to mark today's new playtest build as the first release candidate for our upcoming December release.  We encourage you to give this release a good try out, especially if you like to play competitively online, because if we don't hear your balance complaints now then you will be stuck with them for the next few months until our next release!  You can report any issues you have by [filing a bug](http://bugs.openra.net) or [joining our IRC channel](/community/).
+
+This release fixes several general issues, including:
+
+* Fixed a hang when exiting the game under Windows
+* Fixed an exploit that allowed the color validator to be bypassed
+* Fixed Battlefield News never updating for some players
+* Improved polish and integration of the global chat channel
+
+Fixes for the Dune 2000 mod include:
+
+* Fixed repair depots not repairing units, and buffed repair rate
+* Fixed missing unit crates
+* Fixed incorrect unit armor / damage statistics
+* Improved behaviour of Spice Blooms
+* Improved behaviour of Thumpers and Worms
+* Tweaked support power timers and Ornithoper Strike damage
+
+We further iterated the Red Alert mod balance:
+
+* Fixed units not targeting enemy units under the fog when GPS is available
+* Reduced Spy Power Plant infiltration power-outage time to 20 seconds
+* Improved the speed, sight, and passenger count of England's Phase Tank
+* Improved the cooldown between jumps for Germany's Chrono Tank
+* Improved Russia's Tesla Tank by allowing them to attack while moving
+* Fixed M.A.D. Tanks destroying trees
+* Fixed several issues in the single player missions
+
+The Tiberian Dawn mod recieved some fixes too:
+
+* Fixed Visceroids spawning when infantry are crushed
+* Fixed issues with the C&C64 - GDI01 and Nod05 missions
+* Improved the Nod delivery plane landing animation
+
+As usual, you can find the installer for your operating system on our [download page](/download/)!
+
+
+<div style="text-align:center" markdown="1">
+![Editor copy/paste](/images/news/20151114-copypaste.png)
+
+Suprise bonus feature! Copy/paste is now available in the map editor.
+</div>
+
+<hr />
+
+The second Big Team Tournament concluded yesterday, and was a huge success.  Two teams of 11 players battled over the last week to a 30—30 tie.  The tie-breaker was resolved with a final battle between the two team captains, which ended with a decisive victory for Team Kyrylo!
```

Comment:
It's not my native language, but does a tie-breaker get resolved? Sounds strange to my ears. I'd probably say "The tie-breaker was a match between..." or "The tie was resolved by a match..."